packages/migration/runMigrations.sh

```diff
@@ -9,14 +9,32 @@
 # Copyright (C) The OpenCRVS Authors. OpenCRVS and the OpenCRVS
 # graphic logo are (registered/a) trademark(s) of Plan International.
 
+if [ "$1" != "" ]; then
```

Comment:
@Zangetsu101 when yarn dev runs, the path to core is not passed to the script.  This is only used when called from the Farajaland repo